pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 79
+target-version = ['py37']
+skip-string-normalization = 1
```

Comment:
I didn't modified all single string quotes to double quotes, may be later.

Reply:
chao to double quotes, single quotes should be default

Reply:
except maybe doc strings

Reply:
> chao to double quotes, single quotes should be default

you should look at  https://github.com/MrMrRobat/blacked - it's black with single quiotes, but needs a support for some issues.